### ISSUE TITLE
Vulnerabilities in the azurelinux-3.0-net8.0-webassembly-amd64 image

### ISSUE THREAD

**Issue by MichaelSimons** [dotnet/dotnet-buildtools-prereqs-docker#1281]
- https://github.com/dotnet/dotnet-buildtools-prereqs-docker-internal/issues/253
- https://github.com/dotnet/dotnet-buildtools-prereqs-docker-internal/issues/252
- https://github.com/dotnet/dotnet-buildtools-prereqs-docker-internal/issues/251
- https://github.com/dotnet/dotnet-buildtools-prereqs-docker-internal/issues/250
- https://github.com/dotnet/dotnet-buildtools-prereqs-docker-internal/issues/249
- https://github.com/dotnet/dotnet-buildtools-prereqs-docker-internal/issues/248
- https://github.com/dotnet/dotnet-buildtools-prereqs-docker-internal/issues/247
- https://github.com/dotnet/dotnet-buildtools-prereqs-docker-internal/issues/246
- https://github.com/dotnet/dotnet-buildtools-prereqs-docker-internal/issues/245
- https://github.com/dotnet/dotnet-buildtools-prereqs-docker-internal/issues/244
- https://github.com/dotnet/dotnet-buildtools-prereqs-docker-internal/issues/243
- https://github.com/dotnet/dotnet-buildtools-prereqs-docker-internal/issues/242
- https://github.com/dotnet/dotnet-buildtools-prereqs-docker-internal/issues/241
- https://github.com/dotnet/dotnet-buildtools-prereqs-docker-internal/issues/240
- https://github.com/dotnet/dotnet-buildtools-prereqs-docker-internal/issues/92
- https://github.com/dotnet/dotnet-buildtools-prereqs-docker-internal/issues/82
- https://github.com/dotnet/dotnet-buildtools-prereqs-docker-internal/issues/80

**Comment by MichaelSimons** [comment]
@sbomer - can you take a look at these vulnerabilities and try to address them?  TIA.

**Comment by mthalman** [comment]
No, not without going through a big exception approval process. The proper mitigation here is to update the configuration of the Dockerfile so that it doesn't reference vulnerable versions.
